src/app/SharpRaven/Data/SentryRequestFactory.cs

```diff
@@ -35,7 +35,9 @@
 using System.Linq;
 using System.Reflection;
 using System.Text;
-
+#if net35
+using System.Web;
```

Comment:
Maybe. Could you elaborate as for why System.Web reference might be a problem? Using reflection might be feasible, but there is one method in particular
```csharp
public ISentryRequest Create()
        {
...
}
```

that is expecting the result of **HttpContext** property and it works with the properties of the object (_HttpContext.Request.Url_, _HttpContext.Request.HttpMethod_ and so on) so we would have to use reflection there too, and that implies quite more coding. I'am willing to try, but wouldn't it be better to use System.Web for the moment and then, in another PR, remove it? Again, if you could explain why not using System.Web from the start, I think I'll understand it better.